akd_quorum/src/node/mod.rs

```diff
@@ -0,0 +1,2217 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree and the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree.
+
+//! This module handles the node's primary message handling logic and
```

Comment:
Looks like it handles much more functionality than described here. I'd suggest splitting the mod into smaller pieces each focusing on (ideally) single function with clear mod description - it will make review and future maintainance simpler. 
I've commented down the file on some possible submodules following function naming but this is a just suggestion. What do you think?